README.md

```diff
@@ -4,15 +4,23 @@ An application for [Mattermost](http://mattermost.org) for OS X, Windows, and Li
 
 ![screenshot on OS X](https://cloud.githubusercontent.com/assets/75445/10773661/f6ba012e-7cfc-11e5-9873-e685ea80c860.png)
 
-## Configuring
+## Installation
 
-Create a `config.json` file in the relevant directory for your platform:
+First, clone down this project, and then from within that directory in your favorite terminal run:
 
-- `~/Library/Application Support/Mattermost` on OS X
-- `%APPDATA%/Mattermost` on Windows
-- `~/.config/Mattermost` on Linux
```

Comment:
This is no longer needed with the new install and build system. Everything is run from node when in developer mode, and is run via the executable when built. In both scenarios, all that's required is the `/config.json` file.

This was my primary reason for this build... so that instructions config would be the same on any platform for both test/dev as well as build.

See it in action: https://www.youtube.com/watch?v=zY_eu7UIqBE